CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Convert Closure `Object` to TypeScript `object`.
+- Use glob patterns instead of RegExps to exclude files.
```

Comment:
Recommend adding a `./.github/PULL_REQUEST_TEMPLATE` with a line of:

```
 - [ ] CHANGELOG.md has been updated
```

I find that without it I forget to update the changelog almost all the time.

Reply:
Good idea, done!